StRoot/StMuDSTMaker/COMMON/StMuFstRawHit.cxx

```diff
@@ -0,0 +1,159 @@
+};
+
+unsigned char StMuFstRawHit::getWedge() const
+{
+    return 1 + mGeoId / (kFstNumInnerSensorsPerWedge * kFstNumStripsPerInnerSensor + kFstNumOuterSensorsPerWedge * kFstNumStripsPerOuterSensor);
+};
+
+unsigned char StMuFstRawHit::getPhiStrip() const
+{
+    int strip = mGeoId % (kFstNumInnerSensorsPerWedge * kFstNumStripsPerInnerSensor + kFstNumOuterSensorsPerWedge * kFstNumStripsPerOuterSensor);
+    return strip % kFstNumPhiSegPerWedge;
+};
+
+unsigned char StMuFstRawHit::getRStrip() const
+{
+    int strip = mGeoId % (kFstNumInnerSensorsPerWedge * kFstNumStripsPerInnerSensor + kFstNumOuterSensorsPerWedge * kFstNumStripsPerOuterSensor);
+    return strip / kFstNumPhiSegPerWedge;
+};
+
+unsigned char StMuFstRawHit::getRdo() const
+{
+    return 1 + mChannelId / (kFstNumArmsPerRdo * kFstNumChanPerArm);
+};
+
+unsigned char StMuFstRawHit::getArm() const
+{
+    return (mChannelId % (kFstNumArmsPerRdo * kFstNumChanPerArm)) / kFstNumChanPerArm;
+};
+
+unsigned char StMuFstRawHit::getApv() const
+{
+    return ((mChannelId % (kFstNumArmsPerRdo * kFstNumChanPerArm)) % kFstNumChanPerArm) / kFstNumApvChannels;
+};
+
+unsigned char StMuFstRawHit::getSensor() const
+{
+    int strip = mChannelId % (kFstNumInnerSensorsPerWedge * kFstNumStripsPerInnerSensor + kFstNumOuterSensorsPerWedge * kFstNumStripsPerOuterSensor);
+    if(strip < kFstNumStripsPerInnerSensor) return strip/kFstNumStripsPerInnerSensor;
+    else return strip / kFstNumStripsPerOuterSensor - 1;
+};
+
+unsigned char StMuFstRawHit::getChannel() const
+{
+    return ((mChannelId % (kFstNumArmsPerRdo * kFstNumChanPerArm)) % kFstNumChanPerArm) % kFstNumApvChannels;
+};
+
+float StMuFstRawHit::getCharge( int tb ) const
+{
+    return mCharge[ (tb < 0 || tb >= kFstNumTimeBins) ? mDefaultTimeBin : tb ];
+};
+
+float StMuFstRawHit::getChargeErr( int tb ) const
+{
+    return mChargeErr[ (tb < 0 || tb >= kFstNumTimeBins) ? mDefaultTimeBin : tb ];
+};
+
+//modifiers
+void StMuFstRawHit::setChannelId(int rChannelId)
+{
+    mChannelId = rChannelId;
+};
+
+void StMuFstRawHit::setGeoId(int rGeoId)
+{
+    mGeoId = rGeoId;
+};
+
+void StMuFstRawHit::setSeedhitflag(int rSeedhitflag)
+{
+    mSeedhitflag = rSeedhitflag;
+};
+
+void StMuFstRawHit::setDefaultTimeBin( int tb )
+{
+    mDefaultTimeBin = tb;
+};
+
+void StMuFstRawHit::setIdTruth(unsigned short idTruth)
+{
+    mIdTruth = idTruth;
+};
+
+void StMuFstRawHit::setCharge( float charge, int tb )
+{
+    mCharge[ (tb < 0 || tb >= kFstNumTimeBins) ? mDefaultTimeBin : tb ] = charge;
+};
+
+
+void StMuFstRawHit::setChargeErr(float rChargeErr, int tb)
+{
+    mChargeErr[ (tb < 0 || tb >= kFstNumTimeBins) ? mDefaultTimeBin : tb ] = rChargeErr;
+};
+
+void StMuFstRawHit::setMaxTimeBin(int tb)
+{
+    mMaxTimeBin = ((tb < 0 || tb >= kFstNumTimeBins) ? mDefaultTimeBin : tb);
+};
+
+
+void StMuFstRawHit::print(int nTimeBins)
+{
+    cout << " elecId=" << getChannelId() << " Charge=(" ;
```

Comment:
The STAR coding style suggests to use `LOG_...` streamers instead of std::cout

Reply:
Changed to use LOG_DEBUG instead.